Source/Dafny/DafnyOptions.cs

```diff
@@ -518,6 +519,11 @@ public enum IncludesModes {
           PrintTooltips = true;
           return true;
 
+        case "noParenthesisConstructor":
```

Comment:
What's still missing in DafnyOptions is an additional to the Dafny help command. Look for the text "---- Warning selection" and try to add something similar as what has been done for "warnShadowing". Speaking of "warnShadowing", it might be good to rename "noParenthesisConstructor" to "warnMissingConstuctorParenthesis"